src/crimson/os/seastore/journal.h

```diff
@@ -18,6 +18,11 @@ class NVMeBlockDevice;
 class SegmentManagerGroup;
 class SegmentProvider;
 
+enum class journal_type {
```

Comment:
journal_type_t

Type name are always either CamelCase (generally for complex types) or snake_case_t (simple types like enums).